storage/aptosdb/src/db_debugger/mod.rs

```diff
@@ -30,6 +34,7 @@ impl Cmd {
             Cmd::Checkpoint(cmd) => cmd.run(),
             Cmd::Ledger(cmd) => cmd.run(),
             Cmd::Truncate(cmd) => cmd.run(),
+            Cmd::Examine(cmd) => cmd.run(),
```

Comment:
nit: Cmd::Print?

Reply:
Going to move the state tree subcommand under this, and also add some other subcommands later.